fmn/util.py

```diff
@@ -1,4 +1,5 @@
 import fedora.client
+import fasjson_client
```

Comment:
You should import it as `from . import fasjson_client` or `from .fasjson_client import Client`, otherwise it'll use the system-wide `fasjson_client` if it is installed.